cypress/objects/timelog.js

```diff
@@ -6,7 +6,23 @@ cy.url().should('include', '/time-logging');
 pagetitle(){
 return cy.get('.page__title')
 }
-
+calendarv(){
+return cy.get('.calendar')
+}
+getTile(){
+    return cy.get('.tile.form');
+}
+getUserTitle(){
+    return cy.get('.user-info__title');
+}
+mainNav(){
+    return cy.get('.main-nav').find('li');
+}
+celendarToday(){
+    return cy.get('.calendar--today');
+}
+selectedTimelogging(){
```

Comment:
This does not get the time logging tab, it retrieves the _selected_ tab, so better rename this to `getSelectedTab()` or something :)